package.json

```diff
@@ -4,19 +4,19 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "start": "nodemon src/index.js"
```

Comment:
nodemon is for development purposes. It watches for changes and updates the running server as you work. rename this script to start-dev and add another called "start": "node srx/index.js"